server/initializers/formActions.js

```diff
@@ -169,7 +170,10 @@ function generateFormActions (form) {
     context: {},
     latitude: {},
     longitude: {},
-    radius: {}
+    radius: {},
+    threat: {
+      formatter: inputHelpers.formatter.nomenclatureFilter
+    }
```

Comment:
This will add `threat` parameter to all forms, but some don't have such field. Should be conditional.